src/components/MarkdownImage.tsx

```diff
@@ -42,20 +41,25 @@ const MarkdownImage = ({
   return (
     // display the wrapper as a `span` to avoid dom nesting warnings as mdx
     // sometimes wraps images in `p` tags
-    <Flex as="span" justify="center">
-      <Link href={transformedSrc} target="_blank" rel="noopener" locale={false}>
-        <Image
+    <span className="flex justify-center">
+      <InlineLink
+        href={transformedSrc}
+        target="_blank"
+        rel="noopener"
+        locale={false}
+      >
```

Comment:
I'm not sure we need the `target` and `rel` any more.... we're using `ui/Link` now which should parse this with `const isExternal = url.isExternal(href)`

```suggestion
      <InlineLink href={transformedSrc} locale={false}>
```

Reply:
Never mind! I see we're feeding local images to this so this override is needed. Approving =)